spec/calendar.html

```diff
@@ -439,65 +469,125 @@ <h1>
         <dd>It calls the given _calendar_'s `inLeapYear()` method and validates the result.</dd>
       </dl>
       <emu-alg>
-        1. NOTE: If _calendar_ is a built-in object, then the following step will usually invoke <emu-xref href="#sec-temporal.calendar.prototype.inleapyear"></emu-xref> or <emu-xref href="#sup-temporal.calendar.prototype.inleapyear"></emu-xref>.
+        1. If _calendar_ is a String, then
+          1. Set _calendar_ to ! CreateTemporalCalendar(_calendar_).
+          1. Return ? Call(%Temporal.Calendar.prototype.inLeapYear%, _calendar_, « _dateLike_ »).
         1. Let _result_ be ? Invoke(_calendar_, *"inLeapYear"*, « _dateLike_ »).
         1. If Type(_result_) is not Boolean, throw a *TypeError* exception.
         1. Return _result_.
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-totemporalcalendar" aoid="ToTemporalCalendar">
-      <h1>ToTemporalCalendar ( _temporalCalendarLike_ )</h1>
+    <emu-clause id="sec-temporal-totemporalcalendarslotvalue" type="abstract operation">
+      <h1>
+        ToTemporalCalendarSlotValue (
+          _temporalCalendarLike_: an ECMAScript value,
+        ): either a normal completion containing either a String or an Object, or an abrupt completion
```

Comment:
The new structured headers can align a little better with the prevailing style in ECMA-262.
```suggestion
        ): either a normal completion containing either a String or an Object, or a throw completion
```